src/pineko/scaffold.py

```diff
@@ -0,0 +1,67 @@
+"""Tools related to generation and managing of a pineko project."""
+
+import os
+import pathlib
+
+from .configs import NEEDED_FILES, NEEDED_KEYS
+
+
+def set_up_project(configs):
+    """Set up all the folders spelled out in the configs dictionary.
+
+    Parameters
+    ----------
+    configs : dict
+        configs dictionary containing all the paths to be set up
+    """
+    for path in configs["paths"]:
```

Comment:
Here as well, you are using `configs["paths"][path]` many times.
For readability (but in general also for performance) use the `.items()` iterator (i.e. assign it to a variable, but this way is more idiomatic).